### ISSUE TITLE
Feature Request: line_length ignore "localize(d)" lines.

### ISSUE THREAD

**Issue by fbartho** [realm/SwiftLint#1769]
`NSLocalizedString` & family of methods / macros, and custom-prefixed versions of those can easily get very long, and it doesn't usually provide additional value to the readability of the code to force-wrapping on them.

Proposals
A. provide an option like "ignores_localizable_content" that will ignore lines that contain the substring "localize" (case insensitive). 
B. provide an option like "replacement_regexp_for_length_testing" -- if the user supplies this, then they can cause that regexp be used to modify overly long lines and see if they still hit the limits.
C. Both options A & B

Option A. has the benefit of being comparatively easy to implement, but can lead to permitting ridiculously easy to have long lines because there was a localized string as one of the args.
Option B. solves for the problem in A, as well as being more flexible for large codebases, but requires user education.
Option C. can be implemented by implementing A on top of option B.

**Comment by Noobish1** [comment]
I'd like this also, i'd only really need option A

**Comment by AnthonyMillerSF** [comment]
I would really appreciate this as well!

**Comment by fbartho** [comment]
This feature request is still legitimate!

**Comment by casperson** [comment]
Yeah, really don't like to discount the whole rule due to localized strings.
